dist/yellow-text.js

```diff
@@ -252,6 +247,21 @@
 
       /**
       *
+      * hasContentChanged
```

Comment:
I would suggest renaming the function name to: `contentHasChanged()` to trigger the change. It seems to be a bit confusing to me to have a `isContentChanged` and an `hasContentChanged` method. The `contentHasChanged` describes the function better.